guardian/media-atom-maker
# Add presence indicators to video components on videos page

## What does this change?

This adds presence indicators to the front 'videos' page. Users will now be able to see if anyone's editing a video before they enter the video, which will help prevent problems caused by multiple users being on the same video.

## How to test

It's probably easiest to deploy to CODE so that you don't need to worry about worrying `presence-indicators` locally. Once you're in a environment where this branch is running:
- Enter multiple videos in multiple tabs. Visit the front page in another tab - do your initials show as present in those videos?
- Close those tabs. Are you no longer present?

![Presence indicators on videos page](https://github.com/guardian/media-atom-maker/assets/34686302/acbee097-5d59-4e50-9006-44d1f6f65032)
